<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawReadsVisWidget</class>
 <widget class="QWidget" name="RawReadsVisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="wAccelTop" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="wGyroRight" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="wAccelBack" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="wGyroBack" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="wMagTop" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lbAccelZ">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QWidget" name="wMagRight" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="lbMagY">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="lbMagZ">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lbGyroY">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbGyroX">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="wAccelRight" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QWidget" name="wMagBack" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbMagX">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbAccelY">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lbGyroZ">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="wGyroTop" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbAccelX">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
